### PR TITLE
copying bank transfer data for Polish language

### PR DESCRIPTION
**Pull request by tomasztaw**

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -388,3 +388,24 @@ footer .copy-btn {
 footer .copy-btn:hover {
   background-color: #2c2c00;
 }
+
+/* I don't know if this is right place for this styles */
```
Comment (kavabata) [inline css/global.css]: better to move it here https://github.com/kavabata/sila/blob/42f30ff63e3007b7ac7e02781a73f90657f9ce2b/css/sections.css#L73-L81

File: css/global.css [hunk 2]
```diff
@@ -388,3 +388,24 @@ footer .copy-btn {
...
+.bankDetails {
+  background-color: azure;
+  border-radius: 20px;
```
Comment (kavabata) [inline css/global.css]: use var(--border-radius-lg)

File: css/global.css [hunk 3]
```diff
@@ -388,3 +388,24 @@ footer .copy-btn {
...
+  padding: 20px;
```
Comment (kavabata) [inline css/global.css]: use var(--padding)

File: css/global.css [hunk 4]
```diff
@@ -388,3 +388,24 @@ footer .copy-btn {
...
+  margin: 0 0 15px 5px;
```
Comment (kavabata) [inline css/global.css]: use --padding and --padding-half

File: css/global.css [hunk 5]
```diff
@@ -388,3 +388,24 @@ footer .copy-btn {
...
+}
+
+.copy-bank-btn {
```
Comment (kavabata) [inline css/global.css]: name it .button-copy and place below .button

File: page.php [hunk 6]
```diff
@@ -13,44 +13,98 @@
...
 
   </section>
 </div>
 
-<?php include('templates/footer.php'); ?>
\ No newline at end of file
+<?php include('templates/footer.php'); ?>
+
+<script>
+function copyToClipboard(id) {
```
Comment (kavabata) [inline page.php]: replace function on footer with this one
and let's move it to /js/functions.js and include on header.php

File: page.php [hunk 7]
```diff
@@ -13,44 +13,98 @@
...
               ' . $link['text'] . '</a>';
             endforeach; ?>
+        </div>
+        <?php endif;
+
+        if (isset($page['bank_details'])) : ?>
+        <div class="bankDetails">
+          <div>
+            <strong>Bank:</strong> <span id="bank_name"> <?= $page['bank_details']['bank_name']; ?> </span>
+            <button class="copy-bank-btn" onclick="copyToClipboard('bank_name')" title="copy">
```
Comment (kavabata) [inline page.php]: use more descriptive titles like 'copy bank name to clipboard'

Comment (tomasztaw) [reply page.php]: "bank name" or "bank details"?

Comment (kavabata) [reply page.php]: each button have own title, copy name, copy acc num, copy address etc

**Comment by kavabata** [comment]
Generic name for function like `copyToClipboard` is more preferable.
Update copy button's titles to be more descriptive, this help for screen-readers to better understand what they are doing.
Please add bank_details for en and ua versions, it shouldn't be to complicated.

File: lang/ua.php [hunk 8]
```diff
@@ -112,7 +112,13 @@
...
       'intro' => [
         0 => 'Кожна грошова пожертва на рахунок нашого фонду допомагає нам досягати наших статутних цілей.',
-        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в банку PKO BP<br>номер рахунку: <b>' . $site_account . '</b><br>з назвою: <b>“Пожертва на статутні цілі”</b>'
+        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в'
+      ],
+      'bank_details' => [
+        'bank_name' => 'PKO Bank Polski S.A.',
+        'account_number' => $site_account,
+        'foundation_name' => 'Фонд "Siła Kobiety"',
+        'payment_title' => '"Пожертва на статутні цілі"',
```
Comment (kavabata) [inline lang/ua.php]: podwójne zawiasy

File: page.php [hunk 9]
```diff
@@ -33,28 +33,28 @@
         <div class="bankDetails">
           <div>
             <strong>Bank:</strong> <span id="bank_name"> <?= $page['bank_details']['bank_name']; ?> </span>
-            <button class="button-copy-section" onclick="copyBankDetailsToClipboard('bank_name')" title="copy">
+            <button class="button-copy-bank-name" onclick="copyToClipboard('bank_name')" title="copy">
```
Comment (kavabata) [inline page.php]: use same class for all buttons, but different title

File: templates/footer.php [hunk 10]
```diff
@@ -15,7 +15,7 @@
         <strong class="accountNumber" id="accountNumber">
           <?= $site_account; ?>
         </strong>
-        <button onclick="copyBankDetailsToClipboard('accountNumber')" class="button-copy-footer" title="copy">
+        <button onclick="copyToClipboard('accountNumber')" class="button-copy-footer" title="copy">
```
Comment (kavabata) [inline templates/footer.php]: same class here

File: lang/ua.php [hunk 11]
```diff
@@ -112,7 +112,13 @@
       ],
       'intro' => [
         0 => 'Кожна грошова пожертва на рахунок нашого фонду допомагає нам досягати наших статутних цілей.',
-        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в банку PKO BP<br>номер рахунку: <b>' . $site_account . '</b><br>з назвою: <b>“Пожертва на статутні цілі”</b>'
+        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в'
+      ],
+      'bank_details' => [
+        'bank_name' => 'PKO Bank Polski S.A.',
```
Comment (kavabata) [inline lang/ua.php:118]: bank name not so important, more important is address
so add address field to list of account credentials

**Comment by tomasztaw** [comment]
Is there anything else to improve or can I make marge?

**Review by kavabata — APPROVED** [review]

**Pull request merged**